balancer/balancer.go

```diff
@@ -21,6 +21,7 @@ import (
 
 var mistUtilLoadSingleRequestTimeout = 15 * time.Second
 var mistUtilLoadLoopTimeout = 2 * time.Minute
+var mistLocalAddress = "http://127.0.0.1:4242"
```

Comment:
I guess this could not be true; technically MistController could be running on a different port and we could make it configurable.

Reply:
Oh wait, we totally already do. Duh. I'll update to use that.

Reply:
https://github.com/livepeer/catalyst-api/blob/main/main.go#LL57C1-L58C1

Reply:
TY!